releasenotes/README.md

```diff
@@ -12,6 +12,16 @@ If you use [autocomplete](https://developer.salesforce.com/docs/atlas.en-us.sfdx
 
 [Click here for the v50 release notes.](./v50.md)
 
+## March 4, 2021
+
+These changes are in the release candidate plug-in (`salesforcedx@latest-rc`). We plan to include these changes in the next official release. This list isn't final and is subject to change.
+
+
+* FIX: "FILE HAS NO CONTENT" error after authenticating via `auth:jwt:grant` has been fixed (GitHub Issue [#867](https://github.com/forcedotcom/cli/issues/867))
```

Comment:
edit: You no longer get the error `FILE HAS NO CONTENT` when you run the `auth:jwt:grant` command. (github stuff)

Reply:
After a chat, here's what we came up with: "You no longer get the error `FILE HAS NO CONTENT` when you run any command after authenticating to an org with the `auth:jwt:grant` command." (then github stuff)